test/e2e/snaps/test-snap-bip32.spec.js

```diff
@@ -0,0 +1,139 @@
+const { strict: assert } = require('assert');
+const { check } = require('yargs');
+const { withFixtures } = require('../helpers');
+const { TEST_SNAPS_WEBSITE_URL } = require('./enums');
+
+describe('Test Snap update', function () {
+  it('can install an old and then updated version', async function () {
```

Comment:
This is also incorrect, not sure what it should be. 

We may have to change some of the snap's logic as well. See Slack.

Reply:
I fixed that. It was an over-copy from a previous test and I have remedied it